### ISSUE TITLE
Feature Request: Medical mech equipment buffs

### ISSUE THREAD

**Issue by assmops** [d3athrow/vgstation13#27673]
As it stands, the Odysseus and by extension the medical mech modules are totally worthless and never built unless science is bored, medbay has no paramedics, and telesci is either destroyed or otherwise fucked with. 

The mech mounted syringe gun is totally worthless outside of rushing RND tech levels, as it can only inject antitoxin and inaprovaline despite it saying up to 5 reagents can be loaded, must be manually loaded with syringes, and must be manually fired from the exosuit status panel. At that point, why not just make a rapid syringe gun in RND and go nuts with whatever mix you can make?

The mech mounted sleeper is even more useless, as it can only inject inaprovaline.

I propose that the Odysseus is given some care and its equipment made useful, and I'm throwing out some ideas for new medical mech tools too. I wholly expect the new equipment to be thrown under the rug or be drowned in 'when you code it', but the other changes shouldn't be that hard to implement.

**Existing equipment**
-Have the syringe gun synthesize its own syringes like every other mech gun, not just chemicals. 
-Let the mech sleeper be upgraded with RND stock parts
---Alternatively, add a second or even third mech sleeper that are locked behind materials and biological research with expanded chemical lists, IE; they have access to the same chems that normal sleepers have when upgraded with better parts
-Let the Odysseus equip the foam extinguisher that's locked to the firefighter ripley for whatever reason
---Alternatively, give the Odysseus better temperature resist to let operators do rescues in fires

**New equipment ideas**
__Exosuit Mounted Cryocell__
Exactly what it says. Can be manually loaded with cryomix, or set to synthesize cryoxadone. Temperature can be adjusted by the pilot. While in use, has a large drain on the mech's powercell. Would be locked behind research, needing Bluespace 4, Biological 5, and Data 4, and requiring metal, glass, and plasma to build in a fabricator.

__Exosuit Mounted Auto-Doc__
Lets the pilot perform the duties of a paramedic with a switchtool and mobile operating table in safety, without risking the patient dying due to being exposed to space, or a step being interrupted by getting pushed around in the hallway. A person loaded in would be scanned and operated on. Can be loaded with biomass/meat to grow new limbs and organs for reattachment surgery and transplants, or to give the patient blood. Operation time would take slightly longer than a normal operation with standard tools, IE; a normal scalpel rather than the incision manager. The patient would be automatically ejected once no broken bones, internal bleeding, organ damage, or foreign bodies are detected on a body scan, or could be manually ejected at any time.  Would be locked behind research, needing Biological 5, Materials 5, Engineering 5, and Data 4, and requiring metal, glass, plasma, silver, and diamond to build in a fabricator.

__Exosuit Mounted Medical Beam__
~~Shamelessly stolen from~~ Inspired by an existing item on TGcode, this tool would slowly heal all damage on a single mob, akin to Doc's Delight, but **it would not remove foreign bodies like bullet shrapnel, fix broken bones, or cure diseases**. It would quickly drain the mech's powercell, and would require the mob to stay still to work. Would require deep research, needing Biological 5, Materials 10, Power 8, Bluespace 10, Biological 5, Data 4, and Anomaly 6, and requiring metal, plasma, gold, uranium, and phazon to build in a fabricator .
[<featurerequest>][<help>]

**Comment by Gazathor** [comment]
your first issue with this is wanting to touch mechcode, let that demon rest for a long time.

**Comment by DrSnips** [comment]
Do you know about or how the chem scanner feature works for the ody.?

**Comment by Blithering** [comment]
Yeah, what @DrSnips said. You can expand the ody's list of chems by scanning them.
